src/question_view.py

```diff
@@ -15,6 +15,10 @@ def create_question_view(tk, question_frame):
                               font=helvetica_20, bg=Color.LIGHT_GREEN.description, fg=Color.BLACK.description,
                               wraplength=350, justify='left')
     question_label.grid(row=0, column=0, sticky=tk.E)
+    
+    def showAnswer():
```

Comment:
This new method be added outside of the `create_question_view_method()`

Reply:
you're right, I've added it outside `create_question_view_method()`